Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1699,4 +1699,46 @@ Update aborted.</value>
 Update aborted.</value>
     <comment>{0} module name, {1} filename</comment>
   </data>
+  <data name="CodeInspections_ComponentType_ActiveXDesigner" xml:space="preserve">
+    <value>ActiveX Designer</value>
+  </data>
+  <data name="CodeInspections_ComponentType_ClassModule" xml:space="preserve">
+    <value>Class Module</value>
+  </data>
+  <data name="CodeInspections_ComponentType_ComComponent" xml:space="preserve">
+    <value>COM component</value>
+  </data>
+  <data name="CodeInspections_ComponentType_DocObject" xml:space="preserve">
+    <value>Document Object</value>
+  </data>
+  <data name="CodeInspections_ComponentType_Document" xml:space="preserve">
+    <value>Document</value>
```

Comment:
I would probably call it `Document Module`; the `Document` is too broad and easily confused for different meanings.